mobile-widgets/qml/HintsTextEdit.qml

```diff
@@ -16,7 +16,7 @@ TextField {
 	}
 	onFocusChanged: {
 		focus != focus
-		frame.shouldShow = focus
+		frame.shouldShow = false
```

Comment:
I update the message for this one.
Instead of a dropdown menu we have a autocompleting textfield.